appveyor.yml

```diff
@@ -1,53 +0,0 @@
-environment:
```

Comment:
I assume you removed AppVeyor so the builds are faster, please remember to bring it back before merging.